### DISCUSSION TITLE
What happened to `auto_close`?

### DISCUSSION THREAD

**Discussion by mfdorst** [nvim-tree/nvim-tree.lua#2170] [Q&A] [answered]
Various places online show `nvim-tree` as having an `auto_close` option that, when set to `true`, closes the file tree window when the last buffer is closed. What happened to that option?

**Comment by gegoune** [comment, marked as answer]
This functionality was removed. Please see https://github.com/nvim-tree/nvim-tree.lua/wiki/Auto-Close for reasons behind removal and possible solutions users can put in their configuration at their own risk (not supported by nvim-tree).

**Comment by ruperlopez** [comment]
In which part of the configuration should we place the functions mentioned in this link: https://github.com/nvim-tree/nvim-tree.lua/wiki/Auto-Close ?

**Reply by alex-courtis** [reply]
You can add them anywhere in your init.lua, however autocommands are customarily added at the end.

The autocmd / functions will not actually be executed until after vim starts, so should be safe anywhere.

**Reply by ruperlopez** [reply]
Thank you @alex-courtis !
